Assets/Scripts/Achievements/AchievementsManager.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AchievementsManager : MonoBehaviour
+{
+    public ListAchivements achievements;
+
+    //Is it a betetr Approach to create ithis class as singleton Or Should I  give its reference in the Player View Script    ?
```

Comment:
Creating a singleton would be better approach here. Why? Because it will be easy to get access of this class. Also if you have multiple scenes , then you can persist this class as well by using dontdestroyonload method.